### PR TITLE
Added responsive plugin

### PR DESCRIPTION
**Pull request by FinalAngel**
image

### CODE REVIEW

File: aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/responsive.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% load cms_tags %}
+
+<div class="{{ instance.device_breakpoints }}
+    {% if instance.print_breakpoints %}{% if not instance.device_breakpoints %} visible-print{% endif %}{% endif %}
+    {% if instance.classes %} {{ instance.classes }}{% endif %}"
+    {{ instance.attributes_str }}>
+    {% for plugin in instance.child_plugin_instances %}
+        {% render_plugin plugin %}
+    {% endfor %}
+</div>
```
Comment (vxsx) [inline aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/responsive.html]: should be no end of file newline here, as the element can be inline or inline-block

as well as in the beginning

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 2]
```diff
@@ -356,6 +356,188 @@
...
+                }
+
+                // get the dropdown config
+                function getConfig() {
+                    var tmpConfig = 0;
+                    var tmpCls = inputElement.val().split(' ');
+                    var tmpVal = tmpCls[0].split('-');
+
+                    if (tmpVal.length === 3) {
+                        switch(tmpVal[2]) {
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: space before `(`

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 3]
```diff
@@ -356,6 +356,188 @@
...
+                                tmpConfig = 1;
+                                break;
+                            case 'inline':
+                                tmpConfig = 2;
+                                break;
+                            default:
+                                break;
+                        }
+                    // if its longer then 4 it's inline-block
+                    } else if(tmpVal.length === 4) {
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: space before `(`

File: aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js [hunk 4]
```diff
@@ -356,6 +356,188 @@
...
+                    var tmpChoices = [0, 0, 0, 0];
+                    var tmp = inputElement.val();
+
+                    tmp = getCleanCSS(tmp);
+
+                    // loop through items
+                    tmp.forEach(function (item) {
+                        switch(item) {
+                            case 'visible-xs':
+                                tmpChoices[0] = 1;
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/js/base.js]: i'm a bit lost here - what do the numbers represent?

File: aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/responsive.html (hunk 1 above)
Comment (FinalAngel) [inline aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/responsive.html]: done

**Pull request merged**